src/tls13.c

```diff
@@ -8207,6 +8207,48 @@ int wolfSSL_update_keys(WOLFSSL* ssl)
     return ret;
 }
 
+#if !defined(NO_CERTS)
+/* Set whether mutual authentication is required for TLS v1.3 connections.
+ * Server side only.
+ *
+ * ctx  The SSL/TLS CTX object.
+ * req  1 to indicate required and 0 when not.
+ * returns BAD_FUNC_ARG when ctx is NULL, SIDE_ERROR when not a server and
+ * 0 on success.
+ */
+int wolfSSL_CTX_mutual_auth(WOLFSSL_CTX* ctx, int req)
+{
+    if (ctx == NULL || !IsAtLeastTLSv1_3(ctx->method->version))
```

Comment:
@SparkiDev : How difficult would this be to extend to older TLS versions as well? Seems like a desirable server side feature / option.

Reply:
Good idea!
After checking the TLS 1.2 spec, it is an option to send an empty certificate and ignore or complain about it.
I'll change the code.